src/main/java/com/fasterxml/jackson/databind/ser/BeanPropertyWriter.java

```diff
@@ -874,9 +873,9 @@ public void depositSchemaProperty(ObjectNode propertiesNode,
             ser = provider.findValueSerializer(getType(), this);
         }
         boolean isOptional = !isRequired();
-        if (ser instanceof SchemaAware) {
-            schemaNode = ((SchemaAware) ser).getSchema(provider, hint,
-                    isOptional);
+        if (ser instanceof com.fasterxml.jackson.databind.jsonschema.SchemaAware) {
```

Comment:
No, please leave as-is.

Reply:
just revert this change

Reply:
@luozhenyu I appreciate your avoiding warnings, but is this the only way? (I forget if there was another mechanism).